apps/gateway/src/server.ts

```diff
@@ -21,3 +23,11 @@ function gracefulShutdown() {
 
 process.on('SIGTERM', gracefulShutdown)
 process.on('SIGINT', gracefulShutdown)
+
+process.on('uncaughtException', function (err) {
+  captureException(err)
+})
+
+process.on('unhandledRejection', (reason: string) => {
+  captureMessage(reason)
+})
```

Comment:
important otherwise gateway doesn't recover from any unexpected error (hono is a bit crap honestly)